similarity/manage.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import print_function
+from flask.cli import FlaskGroup
+import click
+
+import webserver
+import metrics
+from operations import HybridMetric
+import db
+
+from sqlalchemy import text
+
+PROCESS_BATCH_SIZE = 10000
+
+cli = FlaskGroup(add_default_commands=False, create_app=webserver.create_app_flaskgroup)
+
+
+@cli.command()
+def init_similarity():
```

Comment:
can you add some documentation to all commands for the output of `—help`? If this is a subcommand, we perhaps don’t need similarity in the name - e.g. `python manage.py similarity init` would be fine as a command